tools/resourcedocsgen/README.md

```diff
@@ -1,15 +1,68 @@
 # Resource Docs generator driver
 
-This tool generates the provider schema using `tfgen`, then calls the docs generator tool in `pulumi/pulumi` to use the schema to generate docs.
+This tool calls the docs generator tool in `pulumi/pulumi` which uses the Pulumi schema for a package to generate API (resource) docs.
 
-## Generating docs
+## Installation
 
-Run `make resource_docs`.
+You can install the `resorucedocsgen` tool just like any other Go-based CLI tool:
```

Comment:
```suggestion
You can install the `resourcedocsgen` tool just like any other Go-based CLI tool:
```

Reply:
```suggestion
You can install the `resourcedocsgen` tool just like any other Go-based CLI tool:
```

Reply:
Thanks for catching that, Levi! For some reason GH won't let me commit the suggestion from the UI here. I'll push an update.